packages/web-components/.storybook/preview-head.html

```diff
@@ -0,0 +1,9 @@
+<script id="anchor-polyfill" type="module">
+  document.addEventListener('anchor-polyfill', () => {
+    if (!window.UPDATE_ANCHOR_ON_ANIMATION_FRAME) {
+      window.UPDATE_ANCHOR_ON_ANIMATION_FRAME = true;
+
+      import('https://unpkg.com/@oddbird/css-anchor-positioning@latest');
```

Comment:
I like the event approach. Do you know if this would trigger multiple HTTP round trips for browsers to at least validate their cache? I wonder if we should switch to manually applying polyfill:

```js
document.addEventListener('anchor-polyfill', async () => {
  if (!window.ANCHOR_POLYFILL) {
    const { default: polyfill } = await import('https://unpkg.com/@oddbird/css-anchor-positioning/dist/css-anchor-positioning-fn.js');
    window.ANCHOR_POLYFILL = polyfill;
  }
  window.ANCHOR_POLYFILL(true);
```

But even this is not very efficient (but might be the only option we have for right now), because with each `polyfill()` call, it'll go through the entire DOM to find CSS to polyfill. I hope we can get the more imperative polyfill function somewhat soon: https://github.com/oddbird/css-anchor-positioning/issues/228.

Reply:
@marchbox This code didn't work exactly but making it sync did. As you know, `preview-head.html` changes are a little hard to test but I think this is working and we can iterate forward.

```
<script id="anchor-polyfill" type="module">
  document.addEventListener('anchor-polyfill', () => {
    if (!window.ANCHOR_POLYFILL) {
      window.ANCHOR_POLYFILL = import('https://unpkg.com/@oddbird/css-anchor-positioning@latest');
    }
    window.ANCHOR_POLYFILL(true);
  });
</script>
```

Reply:
This works great for Dropdown as well, but I had to change the URL to `https://unpkg.com/@oddbird/css-anchor-positioning/dist/css-anchor-positioning-fn.js`.

Reply:
Spent a bit more time on this, what you have works but it's equivalent to this:

```js
<script id="anchor-polyfill" type="module">
  document.addEventListener('anchor-polyfill', () => {
    import('https://unpkg.com/@oddbird/css-anchor-positioning@latest');
  });
</script>
```

And for some reason, I couldn't get the function from importing `css-anchor-positioning-fn.js`.

So, what you originally had worked, sorry about that. I also checked the Network tab, doesn't seem to load the polyfill multiple times, seems the browser would just keep triggering the side effect of the import. Still a bit concerned about the perf impact, but it's probably the best we could do for the time being.